pkg/remote/elb_scanner_test.go

```diff
@@ -0,0 +1,126 @@
+package remote
```

Comment:
Can you rename the file `aws_elbv2_scanner_test.go` to distinct from a future `aws_elb_scanner_test.go`.